### DISCUSSION TITLE
Configuring an external diff-display tool (not for merge resolutions)

### DISCUSSION THREAD

**Discussion by BatmanAoD** [jj-vcs/jj#3403] [Q&A] [answered]
Is there a way to configure an external diff display tool, as git does with `diff.external`? I'd like to try using [`difftastic`](https://github.com/Wilfred/difftastic) to render diffs from `jj diff` and `jj log -p`.

As far as I know, difftastic is only for generating and displaying diffs, and cannot be used as a merge tool. So this isn't equivalent to `ui.diff-editor`.

**Comment by martinvonz** [comment, marked as answer]
There's actually an example of using Difftastic in the config docs: https://martinvonz.github.io/jj/v0.15.1/config/#generating-diffs-by-external-command.

**Reply by BatmanAoD** [reply]
Oops, sorry, I did look before asking but I somehow missed that.
